Fix UnityDemo build

zimmy87 opened the pull request:
<!-- Thank you for submitting a pull request! -->
<!-- ⚠️⚠️ Do Not Delete This! pull_request_template ⚠️⚠️ -->
<!-- Please read our contribution guidelines: https://microsoft.github.io/AirSim/CONTRIBUTING/ -->

Fixes: #3276

## About
This change fixes the build break caused by VehicleCompanion.SetDistortionParam referencing a non-existent parameter. This also fixes an issue where the DllImport of InitVehicleManager was missing two parameters that were added to the API in the AirLibWrapper. 

## How Has This Been Tested?
AirLibWrapper was built with this PR applied and it was confirmed that both car and drone mode in the UnityDemo project can launch without issue.

## Screenshots (if appropriate):

zimmy87 committed:
fix UnityDemo build

This fixes the DllImport of InitVehicleManager so that it matches the method signature in the AirLibWrapper

zimmy87 committed:
Merge remote-tracking branch 'upstream/master'

ClarkZha commented:
This fix solves the build problem in master (on Linux 18.04 with Unity 2019.3.12. )

rajat2004 commented on Unity/UnityDemo/Assets/AirSimAssets/Scripts/Utilities/PInvokeWrapper.cs:
```diff
@@ -13,8 +13,9 @@ public static class PInvokeWrapper {
-            IntPtr GetCameraInfo, IntPtr SetCameraPose, IntPtr SetCameraFoV, IntPtr SetSegmentationObjectid, IntPtr GetSegmentationObjectId,
-            IntPtr PrintLogMessage, IntPtr GetTransformFromUnity, IntPtr Reset, IntPtr GetVelocity, IntPtr GetRayCastHit, IntPtr Pause);
+            IntPtr GetCameraInfo, IntPtr SetCameraPose, IntPtr SetCameraFoV, IntPtr setDistortionParam, IntPtr getDistortionParams,
+            IntPtr setSegmentationObjectId, IntPtr getSegmentationObjectId, IntPtr PrintLogMessage, IntPtr GetTransformFromUnity, 
```
All the other method names in Unity seem to follow the same convention of uppercase method name for the APIs defined in WorldSimApi, is it possible to do the same here? Otherwise this will always be a spot of confusion later on

zimmy87 replied:
Good catch! My latest commit updates this PR to align with the naming conventions of the rest of the Unity project

zimmy87 committed:
Response to CR feedback

Fixing the names of various camera distortion APIs to align with the Unity project's naming conventions

zimmy87 committed:
Merge remote-tracking branch 'upstream/master'

zimmy87 committed:
Merge remote-tracking branch 'upstream/master'
